src/dreyfus_fabric_cleanup.erl

```diff
@@ -35,3 +37,41 @@ active_sigs(#doc{body={Fields}}=Doc) ->
          {ok, Index} = dreyfus_index:design_doc_to_index(Doc, IndexName),
          Index#index.sig
      end || IndexName <- IndexNames].
+
+cleanup_local_doc(DbName, ActiveSigs) ->
```

Comment:
Hi Mayya. The function `fabric:all_docs` cannot collect all the local docs because there is an error in function `fold_local_docs` which is used.